main/res/values/dimens.xml

```diff
@@ -42,7 +42,10 @@
 
     <!-- buttons -->
     <dimen name="textSize_buttonsPrimary">@dimen/textSize_detailsSecondary</dimen>
-w
+
+    <!-- icons -->
+    <dimen name="icon_size">42dp</dimen>
```

Comment:
just a minor thing: the icon itself has a size of `30dp` (as defined in the style below), `42dp` is for the whole button, so maybe it should be called something like `iconButtonSize`, and the setting could be in the same group as `textSize_buttonsPrimary`.

Reply:
Will fix that in one of the next PRs...